docs/setup/pi4.md

```diff
@@ -0,0 +1,47 @@
+# Connect Create® 3 to Raspberry Pi® 4 and set up ROS 2 Galactic
+
+1. Download [Ubuntu® Server 20.04 64-bit](https://ubuntu.com/download/raspberry-pi) and write onto a microSD card.
```

Comment:
all points are numbered (1) in the markdown... they render correctly but it's confusing if you read it raw ahah

Reply:
This is proper markdown for ordered lists.